src/main/java/org/prebid/server/spring/config/WebConfiguration.java

```diff
@@ -252,8 +253,9 @@ AmpHandler openrtbAmpHandler(
     }
 
     @Bean
-    StatusHandler statusHandler(@Value("${status-response:#{null}}") String statusResponse) {
-        return new StatusHandler(statusResponse);
+    StatusHandler statusHandler(List<HealthChecker> healthCheckers) {
+        healthCheckers.forEach(HealthChecker::initialize);
```

Comment:
For sure we need to call the initialize() method for those objects which are instance of PeriodicHealthChecker only.